From 806728212932cdf047aa137d0836a45f09bb09e6 Mon Sep 17 00:00:00 2001
From: SSE4 <tomskside@gmail.com>
Date: Sun, 13 Jan 2019 10:07:16 +0700
Subject: [PATCH] - fix MSVC output to use linux line endings

---
 src/m4.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/m4.c b/src/m4.c
index 79946b9..175ce3a 100644
--- a/src/m4.c
+++ b/src/m4.c
@@ -380,6 +380,10 @@ main (int argc, char *const *argv)
   const char *frozen_file_to_read = NULL;
   const char *frozen_file_to_write = NULL;
   const char *macro_sequence = "";
+#ifdef _MSC_VER
+  _setmode(_fileno( stdout ), _O_BINARY);
+  _setmode(_fileno( stderr ), _O_BINARY);
+#endif
 
   set_program_name (argv[0]);
   retcode = EXIT_SUCCESS;
-- 
2.7.4.windows.1

